src/components/Link/Link.tsx

```diff
@@ -75,23 +75,24 @@ export const Link = React.forwardRef<HTMLElement, LinkProps>(function Link(
         const relProp = target === '_blank' && !rel ? 'noopener noreferrer' : rel;
 
         return (
-            // eslint-disable-next-line jsx-a11y/anchor-has-content
             <a
                 {...(extraProps as React.AnchorHTMLAttributes<HTMLAnchorElement>)}
                 {...commonProps}
                 ref={ref as React.Ref<HTMLAnchorElement>}
                 href={href}
                 target={target}
                 rel={relProp}
-            />
+            >
+                {commonProps.children}
+            </a>
         );
     } else {
         return (
             <span
                 {...(extraProps as React.HTMLAttributes<HTMLSpanElement>)}
                 {...commonProps}
                 ref={ref as React.Ref<HTMLSpanElement>}
-                // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+                role="link"
```

Comment:
I'm not sure that it is true

Reply:
argeed

Reply:
And I think we should remove this behaviour in the next major (#600)